test.js

```diff
@@ -0,0 +1,143 @@
+'use strict'
+
+const test = require('ava')
```

Comment:
Why not tap?

Reply:
Aside from the requirement of using their CLI, I like the design of `ava`, just wanted to try it a bit.

Reply:
That runner requirement is because it _always_ transpiles the code being tested. It makes it unnecessarily difficult to debug things if needed.

Personally, I'd rather see consistency across all "official" modules. 🤷‍♂️

Reply:
ava does not transpile the code anymore.